resources/lang/fr/disposable.php

```diff
@@ -3,78 +3,75 @@
     // Common
     'actions'     => 'Actions',
     'airport'     => 'Aéroport',
-    'all'         => 'Voir Tous',
+    'all'         => 'Afficher tout',
     'altn'        => 'Altn',
     'alternate'   => 'Alternatif',
     'close'       => 'Fermer',
-    'darkmode'    => 'Mode Sombre',
+    'darkmode'    => 'Mode Nuit',
     'dest'        => 'Dest',
     'destination' => 'Destination',
-    'fuel_ob'     => 'Carburant à Bord',
-    'latest'      => 'Derniers',
+    'fuel_ob'     => 'Fuel embarqué',
+    'latest'      => 'Récents',
     'map'         => 'Carte',
-    'notes'       => 'Remarques',
```

Comment:
Why does this one disappear ? Remarques is good i think

Reply:
My bad. I have the release version and I see 3 changes were added this and 2 related to market. Totally my mistake as I did check the other projects dates but for this one I didn't, so should not happen.